src/pages/workspace/WorkspaceInvitePage.js

```diff
@@ -83,19 +83,48 @@ function WorkspaceInvitePage(props) {
     const excludedUsers = useMemo(() => PolicyUtils.getIneligibleInvitees(props.policyMembers, props.personalDetails), [props.policyMembers, props.personalDetails]);
 
     useEffect(() => {
-        const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, searchTerm, excludedUsers);
-
-        // Update selectedOptions with the latest personalDetails and policyMembers information
-        const detailsMap = {};
-        _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
-        const newSelectedOptions = [];
-        _.forEach(selectedOptions, (option) => {
-            newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+        let emails = searchTerm.replace(/\s,\s/g, ',').split(',');
+        emails = _.filter(_.map(emails, word => word.trim()), email => email !== '');
+
+        const newUsersToInviteDict = {};
+        const newPersonalDetailsDict = {};
+        const newSelectedOptionsDict = {};
+
+        _.forEach(emails, (email) => {
+            const inviteOptions = OptionsListUtils.getMemberInviteOptions(props.personalDetails, props.betas, email, excludedUsers);
+    
+            // Update selectedOptions with the latest personalDetails and policyMembers information
+            const detailsMap = {};
+            _.forEach(inviteOptions.personalDetails, (detail) => (detailsMap[detail.login] = OptionsListUtils.formatMemberForList(detail)));
+            
+            const newSelectedOptions = [];
+            _.forEach(selectedOptions, (option) => {
+                newSelectedOptions.push(_.has(detailsMap, option.login) ? {...detailsMap[option.login], isSelected: true} : option);
+            });
+        
+            const userToInvite = inviteOptions.userToInvite;
+
+            // Only add the user to the invites list if it is valid
+            if (userToInvite) {    
```

Comment:
```suggestion
            if (userToInvite) {
```

Interesting, I thought we had a linter rule for trailing spaces. Anyway, not going to block on this.

Reply:
Ah, disregard this, I see you just updated :)